scheduler/kubernetes/snaps-cloud.md

```diff
@@ -86,7 +86,7 @@ Spec:
     Kind:  PersistentVolume
     Name:  pvc-f782bf5c-20e7-11e8-931d-0214683e8447
   Portworx Volume:
-    Snapshot Id:  991673881099191762
+    Cloud Snapshot Id:  991673881099191762
```

Comment:
Actually, I meant the value should be something like "687c60d0-73cb-4055-a08f-33c5ab8d4d8e/149813028909420894-125009403033610837-incr"

Reply:
Fixed

Reply:
We can keep the Key as "Snapshot Id". We'll be able to look at the "Snapshot type" to figure out how we should interpret the ID. 
Also, "Snapshot Type" is missing here.